docs/source/admin/traffic_vault.rst

```diff
@@ -79,6 +79,36 @@ Administration of the PostgreSQL database for Traffic Vault
 
 Similar to administering the Traffic Ops database, the :ref:`admin <database-management>` tool should be used for administering the PostgreSQL Traffic Vault backend.
 
+AES Re-Encryption Tool:
+-----------------------
+
+.. program:: reencrypt
+
+app/db/reencrypt/reencrypt
+=====================================================================
+The :program:`app/db/reencrypt/reencrypt` binary is for use to re-encrypt all keys in the Postgres Traffic Vault.
+
+.. note:: For proper resolution of configuration files, it's recommended that this binary be run from the ``app/db/reencrypt`` directory
+
+Usage
+-----
+``./reencrypt [options]``
+
+Options and Arguments
+---------------------
+.. option:: -newKey
```

Comment:
If an option takes an option-argument it should be shown in the `.. option::` directive's argument, and as long options are a GNU extension it should be shown in GNU standard long form (two `-`s) - according to POSIX guidelines, this single option is equivalent to `-n -e -w -K ey` (ambiguity caused by two `e`s). GNU also recommends that long options be hyphen separated (kebab-case rather than camelCase)

e.g.
```rst
.. option:: --new-key PATH

	The file path for the new base64-encoded AES key.
```

I think switching to kebab-case would be an improvement, personally, but that would be a code change and it doesn't really matter much. What's important is that everywhere it's documented it uses the proper `--` to avoid confusion, and indicates that it requires an argument.


Excerpt from [the GNU libc manual, Section 25.1.1](https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html):

> POSIX recommends these conventions for command line arguments. getopt (see Getopt) and argp_parse (see Argp) make it easy to implement them.
>
>>   * Arguments are options if they begin with a hyphen delimiter (‘-’).
>>   * Multiple options may follow a hyphen delimiter in a single token if the options do not take arguments. Thus, ‘-abc’ is equivalent to ‘-a -b -c’.
>>   * Option names are single alphanumeric characters (as for isalnum; see Classification of Characters).
>>   * Certain options require an argument. For example, the ‘-o’ command of the ld command requires an argument—an output file name.
>>   * An option and its argument may or may not appear as separate tokens. (In other words, the whitespace separating them is optional.) Thus, ‘-o foo’ and ‘-ofoo’ are equivalent.
>>   * Options typically precede other non-option arguments.
>>
>>    The implementations of getopt and argp_parse in the GNU C Library normally make it appear as if all the option arguments were specified before all the non-option arguments for the purposes of parsing, even if the user of your program intermixed option and non-option arguments. They do this by reordering the elements of the argv array. This behavior is nonstandard; if you want to suppress it, define the _POSIX_OPTION_ORDER environment variable. See Standard Environment.
>>   * The argument ‘--’ terminates all options; any following arguments are treated as non-option arguments, even if they begin with a hyphen.
>>   * A token consisting of a single hyphen character is interpreted as an ordinary non-option argument. By convention, it is used to specify input from or output to the standard input and output streams.
>>   * Options may be supplied in any order, or appear multiple times. The interpretation is left up to the particular application program. 
>
> GNU adds long options to these conventions. Long options consist of ‘--’ followed by a name made of alphanumeric characters and dashes. Option names are typically one to three words long, with hyphens to separate words. Users can abbreviate the option names as long as the abbreviations are unique.